### PR TITLE
Cluster report e2e test

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
Located in `test` directory by the golang [standards](https://github.com/golang-standards/project-layout/tree/master/test).
Could be run with `make e2e`

Prerequisites:

| ENV variable            |   Expected value                                                                                                  |
|-------------------------|-------------------------------------------------------------------------------------------------------------------|
|    `CPMA_PASSWD`        |    Password for the cluster                                                                           |
|    `CPMA_LOGIN`         |    Login for the cluster                                                                                          |
|    `CPMA_HOSTNAME`      |    Hostname of the stable cluster                                                                                 |
|    `CPMA_CLUSTERNAME`   |    Hostname from the current context of the stable cluster (could be extracted with `oc whoami --show-context`)   |
|    `CPMA_SSHLOGIN`      |    SSH login for the cluster                                                                                      |
|    `CPMA_SSHPORT`       |    SSH port                                                                                                       |
|    `CPMA_SSHPRIVATEKEY` |    Path for the private key                                                                                       |

### CODE REVIEW

**Comment by Danil-Grigorev** [comment]
The current PR is tested with https://github.com/fusor/mig-ci/pull/67

File: test/e2e/e2e.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/fusor/cpma/pkg/transform"
+
```
Comment (gildub) [inline test/e2e/e2e.go]: No space here as same github.com source

**Comment by Danil-Grigorev** [comment]
Testing trigger: rerun test

File: pkg/transform/cluster/cluster.go [hunk 2]
```diff
@@ -1,13 +1,12 @@
 package cluster
 
 import (
-	"time"
-
 	"github.com/fusor/cpma/pkg/api"
 	O7tapiroute "github.com/openshift/api/route/v1"
 	"github.com/sirupsen/logrus"
 	k8sapicore "k8s.io/api/core/v1"
```
Comment (gildub) [inline pkg/transform/cluster/cluster.go]: Please add an empty line here.

**Pull request merged**